### PR TITLE
WIP Pytave CI

### PR DESCRIPTION
**Pull request by cbm755**
works!  @genuinelucifer

### CODE REVIEW

**Comment by genuinelucifer** [comment]
Oh great :D
And there are veryyy few test fails too!! 
.
Could please try it with mtmiller/pytave?
I changed it to use my fork in which I did some changes  (because of octave_config.h errors)...

**Comment by cbm755** [comment]
We'll need to clean all this up, and get the regular python 2.7 and 3.5 popen2 tests back.

Maybe I'll leave it for you to play with: just merge this into your `ci_pytave` branch...

**Comment by genuinelucifer** [comment]
Oh sure. I guess we can have 1 build for pytave now (since the `pytave_ex` branch has only 9 tests failing now) and 2 build for popen2!? (Or in any other way as u would like).
.
I will merge and clean it...

File: .travis.yml [hunk 1]
```diff
@@ -18,10 +14,24 @@ install:
...
   - sudo apt-get install -qq -y octave liboctave-dev;
-  - "pip install sympy==$SYMPY_VER"
+  - sudo apt-get install -qq -y python;
+  - "pip install --user sympy==$SYMPY_VER"
   - if [ "x$DOCTEST" = "xyes" ]; then
         octave -W --no-gui --eval "pkg install -forge doctest";
     fi
+  - if [ "x$PYTAVE" = "xyes" ]; then
+        sudo apt-get install -qq -y libboost-python-dev python-numpy;
+	# TODO: failures here need to make the tests fail
```
Comment (genuinelucifer) [inline .travis.yml]: @cbm755 For this. Should I run something like:
`octave --path=/home/travis/build/cbm755/octsympy/pytave -W --no-gui --eval "sympref ipc pytave; syms x;"`
before this `if` ends? Will it suffice that octave has error? Or something else?

Comment (cbm755) [inline .travis.yml]: I mean if pytave fails to build, then the travis run should give an error.

**Comment by cbm755** [comment]
doesn't seem to work with pytave master :(

**Pull request closed without merging**